console/src/components/UIKit/atoms/Icon/Icon.js

```diff
@@ -2,6 +2,8 @@ import styled from 'styled-components';
 import { color, typography, layout, space } from 'styled-system';
 
 export const StyledIcon = styled.svg`
+  cursor: ${({ cursor }) => (cursor ? 'pointer' : '')};
```

Comment:
This solution limits setting cursor only to `pointer`, but there are also other types (37 if I counted correctly). 

Also, the name `cursor` doesn't suggest what value should be set for `cursor` property and the `Icon` component silently assumes it's `pointer`. 

Possible solutions:
1. Add the possibility to explicitly set `cursor=pointer` for component `Icon`.
2. Simpler, but still limited solution — rename prop name to `pointer`.

Reply:
I renamed it to pointer currently.